lib/chef/knife/bootstrap_windows_winrm.rb

```diff
@@ -91,8 +91,16 @@ def wait_for_remote_response(wait_max_minutes)
           return
         rescue
           retries_left -= 1
+          # $! will contain the current exception object
+          if $!.class.equal?(Errno::ECONNREFUSED)
```

Comment:
Oh, I didn't realize we were doing a blind rescue.

* We should print the exception in the error message on the blind rescue to facilitate doing the right thing when we're rescuing something we shouldn't
* I'm not sure that AllowUnencrypted would cause a Errno::ECONNREFUSED. I would leave out the hint about checking AllowUnencrypted for the Errno::ECONNREFUSED case.

Yeah, this should be something like:

```
  rescue Errno::ECONNREFUSED => e
    ui.error("Connection refused connecting to HOST:PORT")
    raise
  rescue Exception => e  
    # timeout stuff
    ui.error("No response received from remote node after #{elapsed_time_in_minutes(wait_start_time)} minutes, giving up.")
    ui.error("Exception: #{e.message}")
    raise
  end
```